README.md

```diff
@@ -5,45 +5,92 @@
 
-- [OTE SDK](ote_sdk)
-- [OTE CLI](ote_cli)
-- [OTE Algorithms](external)
+---
+
+## Roadmap
+
+### v0.4.0 (4Q22)
+
+- New algorithms
+
+### v1.0.0 (1Q22)
+
+- Installation through PyPI
+  - Package will be renamed as OTX (OpenVINO Training eXtension)
+- CLI update
+  - Update `find` command to find configurations of tasks/algorithms
+  - Introduce `build` command to customize task or model configurations
+  - Automatic algorihm selection for the `train` command using the given input dataset
+- Adaptation of [Datumaro](https://github.com/openvinotoolkit/datumaro) component as a dataset interface
+
+### v1.1.0 (2Q22)
+
+- Structural update
+- Integrate hyper-parameter optimizations
+
+---
+
+## Repository
+
+- Components
+  - [OTE SDK](ote_sdk)
+  - [OTE CLI](ote_cli)
+  - [OTE Algorithms](external)
+- Branches
+  - [develop](https://github.com/openvinotoolkit/training_extensions/tree/develop)
+    - Mainly maintained branch for releasing new features in the future
+  - [misc](https://github.com/openvinotoolkit/training_extensions/tree/misc)
+    - Previously developed models can be found on this branch
+
+---
 
 ## Quick start guide
 
 In order to get started with OpenVINO™ Training Extensions see [the quick-start guide](QUICK_START_GUIDE.md).
 
-## GitHub Repository
-
-The project files can be found in [OpenVINO™ Training Extensions](https://github.com/openvinotoolkit/training_extensions).
-Previously developed models can be found on the [misc branch](https://github.com/openvinotoolkit/training_extensions/tree/misc).
+---
 
 ## License
 
 Deep Learning Deployment Toolkit is licensed under [Apache License Version 2.0](LICENSE).
 By contributing to the project, you agree to the license and copyright terms therein
 and release your contribution under these terms.
 
+---
+
+## Issues / Discussions
+
+Please use [Issues](https://github.com/openvinotoolkit/training_extensions/issues/new/choose) tab for your bug reporting, feature requesting, or any questions.
+
+---
+
 ## Contributing
 
 Please read the [Contribution guide](CONTRIBUTING.md) before starting work on a pull request.
 
+---
+
 ## Known limitations
 
-Training, export, and evaluation scripts for TensorFlow- and most PyTorch-based models from the [misc](#misc) branch are, currently, not production-ready. They serve exploratory purposes and are not validated.
+Training, export, and evaluation scripts for TensorFlow- and most PyTorch-based models from the [misc](https://github.com/openvinotoolkit/training_extensions/tree/misc) branch are, currently, not production-ready. They serve exploratory purposes and are not validated.
 
 ---
 
```

Comment:
@goodsong81 can you check this updates especially "Intro" and "Roadmap" section?